tests/app/v2/notifications/test_post_notifications.py

```diff
@@ -96,13 +96,13 @@ def test_post_sms_notification_uses_inbound_number_as_sender(client, notify_db_s
     notification_id = notifications[0].id
     assert resp_json['id'] == str(notification_id)
     assert resp_json['content']['from_number'] == '1'
+    assert notifications[0].reply_to_text == '1'
```

Comment:
Would be clearer is this was sms_reply_to_text

Reply:
reply_to_test means either SMS sender, email reply to or letter contact